executor_manager/executors/docker/git_cache_volume_manager.py

```
@@ -0,0 +1,648 @@
+        cmd = [
+            "docker",
+            "run",
+            "--rm",
+            "-v",
+            f"{volume_name}:{MOUNT_POINT}:ro",
+            "alpine:latest",
+            "cat",
+            f"{MOUNT_POINT}/{TOUCH_FILE_METADATA}",
+        ]
+
+        result = subprocess.run(cmd, capture_output=True, text=True, timeout=10, check=True)
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)